eva-server/src/test/java/uk/ac/ebi/eva/server/ws/WSTestHelpers.java

```diff
@@ -61,6 +61,13 @@ public static JSONObject testRestTemplateHelperJsonObject(String url, TestRestTe
         return queryResponse.getResponse().get(0).getResult();
     }
 
+    public static ResponseEntity getRestResponse(String url, TestRestTemplate restTemplate) {
+        return restTemplate.exchange(
+                url, HttpMethod.GET, null,
+                new ParameterizedTypeReference<String>() {
```

Comment:
you don't need a ParameterizedTypeReference for a non-generic type. this whole function is equivalent to restTemplate.getForEntity as explained in my other comment